### DISCUSSION TITLE
Qbittorrent and mergerfs, cant initiate downloads

### DISCUSSION THREAD

**Discussion by Petrutu** [trapexit/mergerfs#1313] [Q&A]
Hello! I've just installed qBittorrent on an LXC container (Proxmox), but I'm having trouble starting downloads. I've already changed the option inside qBittorrent from default to POSIX compliant, and my mergerfs line in fstab looks like this: "/mnt/media* /mnt/tank fuse.mergerfs defaults,nonempty,allow_other,use_ino,cache.files=partial,moveonenospc=true,dropcacheonclose=true,minfreespace=150G,fsname=mergerfs,category.create=mfs 0 0". I have "mnt" set up inside the container. If I try to download directly to one HDD (media12tb), it works. My mergerfs settings are: mp0: /mnt/tank, mp=/mnt/media, mp1: /mnt/media12tb, mp/mnt/media12tb. Can someone please help me troubleshoot this? Thanks!

For installing qBittorrent, I used this script:

bash
Copy code
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/qbittorrent.sh)"
If I try to manually write something to the pool, it works. Thanks!

**Comment by trapexit** [comment]
1) defaults, nonempty, use_ino... those are all deprecated for some time. What version of mergerfs are you using? You should be using the latest release.
2) https://github.com/trapexit/mergerfs#support need more information to help

**Comment by Petrutu** [comment]
Hi @trapexit and thanks for your prompt response this is version currently i have  mergerfs v2.40.1 i will use mergerfs 99% of the time just for torrenting and sonarr radarr etc. do i need to take out defaults, nonempty, use_ino..?  my set up is proxmox and then lxc container. if i must change something on the fstab, can you please point me to the right choises please? Thanks

**Reply by trapexit** [reply]
All the details for such things are in the docs. defaults, nonempty, and use_ino are completely inert. They are just taking up space in your options.

**Reply by Petrutu** [reply]
/mnt/media* /mnt/tank fuse.mergerfs allow_other,cache.files=partial,moveonenospc=true,dropcacheonclose=true,minfreespace=150G,fsname=mergerfs,category.create=mfs 0 0 i do systemctl daemon-reload after i change line in fstab, i reboot the qbittorrent container, but when i start one new download, i got error again  :(
in qbittorrent i have 
Disk IO type (libtorrent >= 2.0; requires restart): (?) | Default                         Memory mapped files                         POSIX-compliant
-- | --
Disk IO read mode                         Enable OS cache
Disk IO write mode:                         Enable OS cache                         Write-through (requires libtorrent >= 2.0.6)

**Reply by trapexit** [reply]
I wasn't claiming those were the issue. I was telling you they have no effect, deprecated, and just should be removed.

**Reply by trapexit** [reply]
If you set `cache.files=auto-full` then it doesn't matter what your IO mode is. If you set it `=off` then it should be set to non-mmap. Again, this would be easier if you look at the logs of qbittorrent for some error messages.

**Comment by Petrutu** [comment]
root@pve:~# uname -a
Linux pve 6.5.13-1-pve #1 SMP PREEMPT_DYNAMIC PMX 6.5.13-1 (2024-02-05T13:50Z) x86_64 GNU/Linux
root@pve:~# lsb_release -a
-bash: lsb_release: command not found
root@pve:~# df -h
Filesystem            Size  Used Avail Use% Mounted on
udev                  7.7G     0  7.7G   0% /dev
tmpfs                 1.6G  1.3M  1.6G   1% /run
/dev/mapper/pve-root   37G  4.0G   31G  12% /
tmpfs                 7.7G   34M  7.7G   1% /dev/shm
tmpfs                 5.0M     0  5.0M   0% /run/lock
efivarfs              128K   63K   61K  51% /sys/firmware/efi/efivars
mergerfs               38T   24T   15T  62% /mnt/tank
/dev/sdc1             5.5T  2.2T  3.4T  40% /mnt/media6tb
/dev/sdf1              11T  7.6T  3.3T  70% /mnt/media12tb
/dev/sde1             9.1T  6.5T  2.6T  72% /mnt/media9.1tb
/dev/sda2            1022M   12M 1011M   2% /boot/efi
/dev/sdg              440G   26G  391G   7% /mnt/appdata
/dev/sdd1             9.1T  5.9T  3.2T  65% /mnt/media9tb
/dev/sdb1             3.6T  1.4T  2.3T  38% /mnt/media3tb
/dev/fuse             128M   20K  128M   1% /etc/pve
tmpfs                 1.6G     0  1.6G   0% /run/user/0

**Reply by trapexit** [reply]
Need details about the app. That's what's you're having problems with. Have you looked at it's logs? Taken an strace like requested in the support docs? Tried a minimal setup? Shown what your container config is? Checked the entitlements?
